src/dataframe/dataframe.jl

```diff
@@ -589,12 +580,14 @@ end
 ##############################################################################
 
 """
-    insertcols!(df::DataFrame, ind::Int; name=col,
-                makeunique::Bool=false)
-    insertcols!(df::DataFrame, ind::Int, (:name => col)::Pair{Symbol,<:AbstractVector};
-                makeunique::Bool=false)
+    insertcols!(df::DataFrame, ind::Int, (name=>col)::Pair{Symbol,<:AbstractVector}...;
```

Comment:
With this you could avoid repeating the signature. Maybe that's clearer?
```suggestion
    insertcols!(df::DataFrame, [ind::Int], (name=>col)::Pair{Symbol,<:AbstractVector}...;
```